ThreeBotPackages/zerobot/admin/frontend_src/sources/views/main.js

```diff
@@ -118,6 +118,9 @@ export default class TopView extends JetView {
             }, {
                 id: "deployed_gateway_4to6",
                 value: '<span><img class="solutions-icon" src="static/img/ip.png"/>4 to 6 Gateway</span>'
+            }, {
+                id: "deployed_monitoring",
+                value: '<span><img class="solutions-icon" src="static/img/redis.png"/>Monitoring</span>'
```

Comment:
Can we pick something else then the redis logo as a icon for the monitoring chat flow. Redis has 0 connotation with monitoring this is confusing.

Reply:
Sure, I can look for something more generic like a simple graph?

Reply:
sounds good

Reply:
updated to this https://github.com/threefoldtech/jumpscaleX_threebot/blob/2c57640372d6d533176a56676396ab87810b0f3b/ThreeBotPackages/zerobot/admin/frontend_src/static/img/monitoring_graph.png